### DISCUSSION TITLE
When should developers use whisper.cpp vs faster-whisper?

### DISCUSSION THREAD

**Discussion by chen-rn** [SYSTRAN/faster-whisper#496] [Q&A]
After reading through a few different GitHub issues, it seems that faster-whisper is **drastically** faster than whisper.cpp with similar output. For both CPU and GPU. There doesn't seem to be any downside. 

I'm thinking of using faster-whisper for a new project, but I'm afraid I'm missing something obvious here.

**Comment by phineas-pta** [comment]
whisper.cpp = cross-platform & cross-language = deploy on iphone android raspberry, in java js rust etc.

faster-whisper = u stick with python, on pc with with decent hardware, ofc it's more optimized

**Comment by phamkhactu** [comment]
Hi @phineas-pta,

Have you ever converted whisper model to ggml or guff ? And if I want to set lang in converted model(ggml) how to set it?

**Reply by phineas-pta** [reply]
ggml đâu có lang, bác chọn lang lúc chạy `whisper.cpp` mà

**Reply by phamkhactu** [reply]
> ggml đâu có lang, bác chọn lang lúc chạy `whisper.cpp` mà

@phineas-pta Thực ra mình muốn đưa whisper về realtime, cái chính khi đưa về near realtime để chạy on edge thì sẽ chỉ là ggml chưa chỉ định rõ được ```lang``` thế nên mình đang hỏi để confirm điều này ạ? Bạn đã thử biến whisper về near realtime or realtime chưa?

**Reply by phineas-pta** [reply]
e chưa làm on edge nên ko chắc 

nhưng chắc chắn 1 điều là ggml ko có `lang`, nó chỉ là 1 file thôi, bác chọn `lang` lúc chạy, nó trong các argument mà
